### PR TITLE
KOGITO-2298: Renamed Process list related components

### PR DESCRIPTION
**Pull request by Sara4994**
Hello! Please review, this PR contains changes to name of few react components and these modifications are intended to meet a standard and uniformity on components across the packages.

JIRA issue: https://issues.redhat.com/browse/KOGITO-2298

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@Sara4994 overall looks ok, but I would like to wait for https://github.com/kiegroup/kogito-apps/pull/256 to be merged first as we do have some overlapping changes.

**Comment by Sara4994** [comment]
> @Sara4994 overall looks ok, but I would like to wait for #256 to be merged first as we do have some overlapping changes.

@cristianonicolai  i noticed PR #256 a bit late... but it will be good if "removing components string" is added as part of this PR, because they were all used and related to process list page... So we can have just one PR to do all those changes.. wdyt?

**Comment by cristianonicolai** [comment]
> > @Sara4994 overall looks ok, but I would like to wait for #256 to be merged first as we do have some overlapping changes.
> 
> @cristianonicolai i noticed PR #256 a bit late... but it will be good if "removing components string" is added as part of this PR, because they were all used and related to process list page... So we can have just one PR to do all those changes.. wdyt?

thats fine, I can rebase my PR once this one is merged

**Review by cristianonicolai — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]
everything working well :+1:

**Review by jstastny-cz — APPROVED** [review]

**Comment by cristianonicolai** [comment]
@Sara4994 needs rebase

File: packages/task-console/src/components/Templates/BaseComponent/BaseComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
```
Comment (cristianonicolai) [inline packages/task-console/src/components/Templates/BaseComponent/BaseComponent.tsx]: @Sara4994 needs to be removed

**Review by nmirasch — APPROVED** [review]

File: packages/task-console/src/components/Templates/BaseComponent/BaseComponent.tsx (hunk 1 above)
Comment (Sara4994) [reply packages/task-console/src/components/Templates/BaseComponent/BaseComponent.tsx]: done

**Pull request merged**